packages/sample-project/cannonfile.toml

```diff
@@ -2,20 +2,22 @@ name = "synthetix-sample-project"
 version = "<%= package.version %>"
 description = "Sample usage of the router manager"
 
-# `salt` here only affects proxy contract
 [setting.salt]
 description = "Change this to a unique string when deploying multiple instances of the system. Note that only the proxy contract will be unique."
-defaultValue = "main"
+defaultValue = "sample-project"
 
 [setting.owner]
 description = "Admin user for the system"
 defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
 
-[contract.OwnerModule]
-artifact = "contracts/modules/OwnerModule.sol:OwnerModule"
+[setting.core_router_version]
+description = "Version of @synthetixio/core-router"
+defaultValue = "1.0.0"
 
-[contract.UpgradeModule]
-artifact = "contracts/modules/UpgradeModule.sol:UpgradeModule"
+[import.core_router]
+source = "core-router:<%= settings.core_router_version %>"
```

Comment:
would be better to have setting for `[setting.core_router_package]` and inject the whole package name in here rather than individual part